lib/helpers/getNParents.js

```diff
@@ -0,0 +1,18 @@
+import { getDomNodeProfile } from './getDomNodeProfile';
+
+export function getNParents(element, nParents = 5, parents = []) {
+  let parent = element.parentNode;
+
+  if (nParents === 0 || !parent) {
+    return parents;
+  }
+
+  return getNParents(
+    parent,
+    nParents - 1,
+    parents.concat({
+      ...getDomNodeProfile(parent, { basicProfile: true }),
```

Comment:
Instead of using options flags to change behavior of the method, why not just pull basic profile code into a new method?